### PR TITLE
Update to latest information

### PR DESCRIPTION
**Pull request by bcrochet**
Reconcile the document with the current implementation details.

Signed-off-by: Brad P. Crochet <brad@redhat.com>

### CODE REVIEW

File: enhancements/kubernetes-nmstate-integration.md [hunk 1]
```diff
@@ -124,6 +144,8 @@ Initial support for kubernetes-nmstate will be Tech Preview
 
 #### Tech Preview -> GA
 
+- kubernetes-nmstate handler can be installed by kubernetes-nmstate-operator
```
Comment (phoracek) [inline enhancements/kubernetes-nmstate-integration.md]: How is this related to the graduation? We need the operator to be able to install knmstate in OCP even in TP.

File: enhancements/kubernetes-nmstate-integration.md [hunk 2]
```diff
@@ -57,15 +57,25 @@ what this project is addressing.
...
+A new kubernetes-nmstate handler DaemonSet is deployed in the cluster by the operator.
 This DaemonSet contains nmstate package and interacts with the NetworkManager
-on the host by mounting the related dbus. The project contains two
-Custom Resource Definitions, `NodeNetworkState` and `NodeNetworkConfigurationPolicy`.
-`NodeNetworkState` objects are created per each node in the cluster and can be
-used to report available interfaces and network configuration. These objects
-are created by kubernetes-nmstate and must not be touched by a user.
+on the host by mounting the related dbus. The project contains three
+Custom Resource Definitions, `NodeNetworkState`, `NodeNetworkConfigurationPolicy` and
+`NodeNetworkConfigurationEnactment`. `NodeNetworkState` objects are created per each
```
Comment (qinqon) [inline enhancements/kubernetes-nmstate-integration.md:72]: Dup

**Review by qinqon — CHANGES_REQUESTED** [review]
Nit

File: enhancements/kubernetes-nmstate-integration.md [hunk 3]
```diff
@@ -57,15 +57,25 @@ what this project is addressing.
 
 ## Proposal
 
-A new kubernetes-nmstate handler DaemonSet is deployed in the cluster part of the OpenShift installation.
+A new kubernetes-nmstate operator Deployment is deployed to the cluster as part of the the OpenShift
+installation, via CVO.
+
+The operator has a single CRD, called `NMstate`. When a custom resource of that type is created,
```
Comment (SchSeba) [inline enhancements/kubernetes-nmstate-integration.md]: "When a custom resource of that type is created"

It should have a specific name?
How we handle multiple CRs?

File: enhancements/kubernetes-nmstate-integration.md (hunk 1 above)
Comment (bcrochet) [reply enhancements/kubernetes-nmstate-integration.md]: True. I'll remove.

File: enhancements/kubernetes-nmstate-integration.md (hunk 3 above)
Comment (bcrochet) [reply enhancements/kubernetes-nmstate-integration.md]: I'd say that's a good idea. That we look for a specific name. There only needs to be a single CR. Right now, if you add another (with a different name), it just reconciles again.

File: enhancements/kubernetes-nmstate-integration.md (hunk 2 above)
Comment (bcrochet) [reply enhancements/kubernetes-nmstate-integration.md:72]: I don't understand... What's duplicated?

Comment (qinqon) [reply enhancements/kubernetes-nmstate-integration.md:72]: Is all good, I miss readed it.

**Review by SchSeba — APPROVED** [review]
Thanks!
